ledger/src/blockstore.rs

```diff
@@ -1018,13 +1039,27 @@ impl Blockstore {
             &mut write_batch,
         )?;
 
-        for (erasure_set, erasure_meta) in erasure_metas {
+        for (erasure_set, working_erasure_meta) in erasure_metas {
+            if !working_erasure_meta.should_write() {
+                // No need to rewrite the column
+                continue;
+            }
             let (slot, fec_set_index) = erasure_set.store_key();
-            write_batch.put::<cf::ErasureMeta>((slot, u64::from(fec_set_index)), &erasure_meta)?;
+            write_batch.put::<cf::ErasureMeta>(
+                (slot, u64::from(fec_set_index)),
+                working_erasure_meta.as_ref(),
```

Comment:
I _think_ you can just write `& working_erasure_meta`.
`&` will do the `.as_ref()`.

Reply:
hmm it doesn't seem to work it only recognizes it as a `&WorkingEntry<ErasureMeta>`